frontend/lib/ui/test_results_page/test_results_helpers.dart

```diff
@@ -41,23 +41,36 @@ class TestResultHelpers {
   static void navigateToTestExecution(TestResultWithContext result) {
     final testExecution = result.testExecution;
     final artefact = result.artefact;
+    final testResult = result.testResult;
     final environment = testExecution.environment;
 
     final family = artefact.family;
     final artefactId = artefact.id;
     final testPlan = testExecution.testPlan.trim().isNotEmpty
         ? testExecution.testPlan
-        : 'unknown'; // there exists test results without test plans
+        : 'unknown';
 
     final environmentName = environment.name;
+    final testExecutionId = testExecution.id;
+    final testResultId = testResult.id;
 
     final currentUri = Uri.base;
 
-    final encodedTestPlan = Uri.encodeQueryComponent(testPlan);
-    final encodedEnvironment = Uri.encodeQueryComponent(environmentName);
+    final queryParams = {
+      'Test plan': testPlan,
```

Comment:
Do you think it's still necessary to include "Test plan" and "Environment"? I feel like we can now remove those without any problems.